projects/schematics/package.json

```diff
@@ -13,8 +13,8 @@
   "scripts": {
     "build": "npm run clean && npm --prefix ../../ run generate:deps && ../../node_modules/.bin/tsc -p ./tsconfig.schematics.json",
     "clean": "../../node_modules/.bin/rimraf --glob \"index.js*\" \"index.d.ts\" \"src/**/*.js\" \"src/**/*.js.map\" \"src/**/*.d.ts\" && npm run copy-feature-toggles",
-    "test": "npm run clean && ../../node_modules/.bin/jest --config ./jest.schematics.config.js",
```

Comment:
MAJOR: i suppose these changes are not part of this development. So please revert this.

Reply:
Strangely, I don't see any changes to this file in my local system, neither do I remember committing this file.

Reply:
Found the changed file, reverted